docs/release-notes.rst

```diff
@@ -1,3 +1,199 @@
+Release Notes (Version 2.5)
+===========================
+
+General
+-------
+
+**File system convergence**
+
+APIC Filesystem Glusterfs takes 15 to 20 Minutes to recover when APIC cluster goes unhealthy
+
+During the APIC operations like APIC reboot, upgrade, or decommission/recommission, the APIC filesystem needs some time to recover and resume.
+
+Symptoms users may see:
+1. Application operations throw a (sqlite3.OperationalError) disk I/O error
+2. User may not find previously added BIG-IP entries in the application
+3. New Application installation may fail
+
+**Workaround**: Wait for 20 minutes and try to access the application again. The Glusterfs should recover automatically and the application should be accessible again.
+
+------
+
+**F5 ACI ServiceCenter HA clusters show IPs even if the devices are added using hostnames**
+
+For BIG-IP devices in an HA cluster, when one BIG-IP with hostname is added to F5 ACI ServiceCenter, the IP corresponding to that hostname also gets added in the side menu. The application warns the user about this, and displays a deletion prompt for retaining only the hostname version of the BIG-IP. 
+
+**Workaround:** User may click **Delete** to proceed with deletion of the additional IP version of the device added to the app. OR
+user may skip the prompt and manually delete the device which got added using IP by clicking on the delete (X) icon next to the IP of the device. Repeat the same process for the peer of this device - now both devices should be seen inside the cluster with hostname.
+
+------
+
+**Floating IP auto sync and Default Gateway auto sync will not work when hosts are added in an HA cluster using hostnames**
+
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.X**
+
+User menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB** → **Export DB**, for exporting the F5 ACI ServiceCenter database. This option is not working on APIC versions 5.0.X and higher. This is working as expected on prior versions of the APIC. 
```

Comment:
The User menu...
same comment on the arrow, either change to >  or leave if you know it works
This option does not work on APIC versions...